strongbox-web-core/src/test/java/org/carlspring/strongbox/security/authentication/StrongboxAuthenticationFilterTest.java

```diff
@@ -0,0 +1,107 @@
+package org.carlspring.strongbox.security.authentication;
+
+import org.carlspring.strongbox.security.authentication.strategy.DelegatingAuthenticationConverter;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+import org.springframework.security.authentication.AuthenticationManager;
+import org.springframework.security.core.Authentication;
+
+import javax.servlet.FilterChain;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+class StrongboxAuthenticationFilterTest {
```

Comment:
Cosmetic: Could you please reformat this according to our [coding convention](https://strongbox.github.io/developer-guide/coding-convention.html)? Thanks!

Braces on new line, etc.

Reply:
Done